templates/vm.xml.j2

```diff
@@ -124,6 +124,15 @@
       <listen type='address'/>
     </graphics>
 {% endif %}
+{% for usb_device in usb_devices %}
+    <hostdev mode='subsystem' type='usb' managed='yes'>
+      <source>
+        <vendor id='{{ usb_device.vendor }}'/>
+        <product id='{{ usb_device.product }}'/>
+      </source>
+      <address type='usb' bus='0' port='{{ loop.index }}'/>
```

Comment:
Is it necessary to define the port? In many cases libvirt will fill in missing fields.

Reply:
I will test that and get back to you and adjust accordingly

Reply:
This appears to work fine in testing and handles multiple USB devices fine. As you say libvirt handles the missing information fine and populates this. Whilst not new to virtualisation new to KVM. Thanks for the tip

output of XML dump from vm created with commit https://github.com/stackhpc/ansible-role-libvirt-vm/pull/95/commits/145d3cf23c3307d93cb7258d71f100ed43a884cf
```
    <hostdev mode='subsystem' type='usb' managed='yes'>
      <source>
        <vendor id='0x0781'/>
        <product id='0x5567'/>
        <address bus='1' device='2'/>
      </source>
      <alias name='hostdev0'/>
      <address type='usb' bus='0' port='1'/>
    </hostdev>
    <hostdev mode='subsystem' type='usb' managed='yes'>
      <source>
        <vendor id='0x0930'/>
        <product id='0x6545'/>
        <address bus='3' device='2'/>
      </source>
      <alias name='hostdev1'/>
      <address type='usb' bus='0' port='2'/>
    </hostdev>
    ```